tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -85,38 +54,6 @@ Object.Dai {
 		Object.Base.hw_config."ALH3" {
 			id	1
 		}
-
```

Comment:
line 50: why is the default_hw_confg_id '1'?

Reply:
because the're sonly one hw_config and its ID is 1 on line 55

Reply:
@plbossart do you mean other topologies? We blindly set the hw_config ID to match the link ID in the machine driver for no good reason.

Reply:
We should fix this and make it non ambiguous prior to roll out of topology2. If it not needed we can remove or we should start IDs at 0. We have time to align, @ranj063 are you OK to look at this when convenient ?